src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/data/Watchlist.kt

```diff
@@ -0,0 +1,6 @@
+package uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.data
+
+data class Watchlist(
```

Comment:
You don't need a request body - you can get everything from the path or the header